src/main/java/com/project/trackfit/service/UserService.java

```diff
@@ -0,0 +1,9 @@
+package com.project.trackfit.service;
+
+import com.project.trackfit.exception.UserDoesNotExistException;
+import com.project.trackfit.model.User;
+
+public interface UserService {
+    User createUser(User user);
```

Comment:
```suggestion
    Long createUser(User user);
```

Return the id instead of the user

Reply:
Done